tests/validatingadmissionpolicy/noderestrictions.go

```diff
@@ -193,4 +206,69 @@ var _ = Describe("[sig-compute] virt-handler node restrictions via validatingAdm
 			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("%s should not fail on node specific node restriction", description))
 		}
 	})
+
+	Context("patching another node", func() {
+		BeforeEach(func() {
+			nodesList := libnode.GetAllSchedulableNodes(virtClient)
+			Expect(nodesList.Items).ToNot(BeEmpty())
+			Expect(nodesList.Items).ToNot(HaveLen(1))
+			for _, node := range nodesList.Items {
+				if nodeName != node.Name {
+					anotherNode = node.Name
+					break
+				}
+			}
+
+			prepareNode(anotherNode)
+		})
+
+		AfterEach(func() {
+			cleanup(anotherNode)
+		})
+
+		It("rejects kubevirt related patches", func() {
+			patchSetList := map[string]*patch.PatchSet{
+				"kubevirt.io label addition":      patch.New(patch.WithAdd("/metadata/labels/kubevirt.io~1newAllowedLabel", "value")),
+				"kubevirt.io label update":        patch.New(patch.WithReplace(allowedLabelPath, "other-value")),
+				"kubevirt.io label removal":       patch.New(patch.WithRemove(allowedLabelPath)),
+				"kubevirt.io annotation addition": patch.New(patch.WithAdd("/metadata/annotations/kubevirt.io~1newAllowedAnnotation", "value")),
+				"kubevirt.io annotation update":   patch.New(patch.WithReplace(allowedAnnotationPath, "other-value")),
+				"kubevirt.io annotation removal":  patch.New(patch.WithRemove(allowedAnnotationPath)),
+			}
+			node, err := virtClient.CoreV1().Nodes().Get(context.Background(), nodeName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			pod, err := libnode.GetVirtHandlerPod(virtClient, node.Name)
+			Expect(err).ToNot(HaveOccurred())
+
+			token, err := exec.ExecuteCommandOnPod(
+				pod,
+				"virt-handler",
+				[]string{"cat",
+					"/var/run/secrets/kubernetes.io/serviceaccount/token",
+				},
+			)
+
+			Expect(err).ToNot(HaveOccurred())
+			handlerClient, err := kubecli.GetKubevirtClientFromRESTConfig(&rest.Config{
+				Host: virtClient.Config().Host,
+				TLSClientConfig: rest.TLSClientConfig{
+					Insecure: true,
+				},
+				BearerToken: token,
+			})
+			Expect(err).ToNot(HaveOccurred())
+
+			otherNode, err := virtClient.CoreV1().Nodes().Get(context.Background(), anotherNode, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			for description, patchSet := range patchSetList {
+				nodePatch, err := patchSet.GeneratePayload()
+				Expect(err).ToNot(HaveOccurred())
+				_, err = handlerClient.CoreV1().Nodes().Patch(context.TODO(), otherNode.Name, types.JSONPatchType, nodePatch, metav1.PatchOptions{})
```

Comment:
nit: Unless you plan to replace `context.TODO()` with a concrete context, you better use `context.Background()`.